README.md

```diff
@@ -142,6 +142,6 @@ ${varname:doublequote} => "value1","value2","value3"
 The expected use of multi-value variables is for `in` queries, for example:
 
 ```bash
-$serverHost=(${host:singlequote})
+$serverHost in (${host:singlequote})
```

Comment:
I assume this is graphana rather than Scalyr specific query. What would be advantage to use `in` instead of `=` when I would assume `${host:singlequote}` should return only a single host name, not a collection.

Reply:
It's actually a mix of both `$serverHost in (${host:singlequote})` allows the Grafana `$host` variable (which may be a single or multiple values) to match `$serverHost`:

```
# $host={'foo'}
$serverHost in ('foo')

# $host={'foo','bar'}
$serverHost in ('foo','bar')
```